src/pages/content-only/[...slug].tsx

```diff
@@ -50,7 +50,7 @@ export default renderedPageNoHooks<SlugProps>(({ pageData }) => {
       authorization={pageData.authorization}
     >
       <Script
-        src="/_assets/iframeResizer.contentWindow.min.js"
+        src="iframe-resizer/js/iframeResizer.contentWindow.min.js"
```

Comment:
this will attempt to load https://frontend-git-fix-import-script-for-edusharing-serlo.vercel.app/content-only/iframe-resizer/js/iframeResizer.contentWindow.min.js and fail